app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -83,6 +83,19 @@ class LuciBuildService {
     return getBuilds(slug, sha, builderName, 'try', tags);
   }
 
+  /// Returns an Iterable of try Buildbucket [Build]s for a given [PullRequest].
+  Future<Iterable<Build>> getTryBuildsByPullRequest(
```

Comment:
can we add a coverage test?

Reply:
If you like. The underlying query method is covered but I can add something for this.

Reply:
Done. Added a test for the new method.

Reply:
Does it make sense to have a test where it successfully cancels the builds which are returned by this `getTryBuildsByPullRequest` (containing builds from multiple commits), where `getTryBuilds` is not able to return?

Reply:
I think it would however we are mocking the BuildBucketClient here so it would be just me returning the difference in the lists. I have provided a query in the description where you can see the difference in the data returned for another PR I had opened which needed an update. The sha request only returns the builds for that sh whereas this method returns all of the builds for this pr regardless of commit hash.